collect_stata/write_json.py

```diff
@@ -1,4 +1,5 @@
-"""write_json.py"""
+"""Write calculations and metadata out as a json file.
+"""
```

Comment:
Please keep to the [pep 257 Docstring Conventions](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings)
"One-liners are for really obvious cases. They should really fit on one line."